ktor-client/ktor-client-features/ktor-client-tracing/ktor-client-tracing-stetho/src/androidMain/kotlin/io/ktor/client/features/tracing/KtorInterceptorHeaders.kt

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Consider changing sourceSet dir from `src/androidMain` to `android/src`(as for other sourceSets we have in ktor)

Reply:
Moved.